test/Microsoft.NET.Build.Tests/GivenThatWeWanSatelliteAssembliesHaveassemblyVersion.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using Xunit.Abstractions;
+using System.Diagnostics;
+using FluentAssertions;
+using System.Reflection;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWanSatelliteAssembliesHaveassemblyVersion : SdkTest
```

Comment:
There is a typo here. It should say GivenThatWeWant, it is missing he final t. Also, HaveassemblyVersion should be HaveAssemblyVersion.